packages/ui-extensions/src/surfaces/customer-account/api/docs.ts

```diff
@@ -77,3 +78,48 @@ export interface Docs_Standard_QueryApi
 export interface Docs_StandardApi extends Omit<StandardApi<any>, 'router'> {}
 
 export interface Docs_FullPageApi extends Omit<FullPageApi, 'location'> {}
+
+export interface Docs_Page_Button_PrimaryAction
+  extends Pick<
+    ButtonProps,
+    | 'onPress'
+    | 'to'
+    | 'loading'
+    | 'loadingLabel'
+    | 'disabled'
+    | 'accessibilityLabel'
+  > {}
+export interface Docs_Page_Button_SecondaryAction
+  extends Pick<
+    ButtonProps,
+    'onPress' | 'to' | 'loading' | 'loadingLabel' | 'disabled'
+  > {
+  /**
+   * A label used for buyers using assistive technologies. Needed because `children` passed to this component will be discarded.
+   */
+  accessibilityLabel: ButtonProps['accessibilityLabel'];
```

Comment:
One downside is that this messes with the order of props in the doc. the one you override always come last.

We could do something like:

```ts
export interface Docs_Page_Button_SecondaryAction {
  to: ButtonProps['to']
  onPress: ButtonProps['onPress']
  // etc..
}
```

But then we'd need to copy paste the comment for each prop from the original component over and over again.

Reply:
I'll have a look at adding this to the host so we don't need to bump this package over and over again.